vendor/tcib/nova-compute-arbiterd/nova-compute-arbiterd.yaml

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Copyright 2021 - 2021, Sean Mooney and the arbiterd contributors
+# SPDX-License-Identifier: Apache-2.0
+---
+# we need a base contaienr with libvirtd and the nova user defiend so that
+# file permssions for the nova.conf are correct so use nova-libvirt?
+tcib_from:  quay.io/tripleomastercentos9/openstack-nova-compute:86f3dae907057e946a93f187b256cb6c
+tcib_actions:
+  - user: root
+  - run: adduser -Urm -G libvirt,nova arbiterd
+  - run: mkdir -m 770 /opt/arbiterd
+  - run: dnf install -y python3-pip
+  - run: chown arbiterd:arbiterd /opt/arbiterd
+  - user: arbiterd
+  - run: mkdir -p -m 660 /opt/arbiterd/etc/nova
+  - run: mkdir -p -m 660 /opt/arbiterd/etc/arbiterd
+  - run: pip install arbiterd
+tcib_user: arbiterd
+tcib_entrypoint: arbiterd-static --nova-config /opt/arbiterd/etc/nova/nova.conf --arbitrate cpu-state
```

Comment:
ok so the nova-compute-arbiterd contaienr does not need to do most of the above.

this container does not need to have arbiterd installed. 
it just need to have this patch to nova applied.

git fetch https://review.opendev.org/openstack/nova refs/changes/28/821228/1 && git format-patch -1 --stdout FETCH_HEAD

I'm not entirely sure what the best way to do that is.
we could download the srouce rpm and then add this patch and rebuild and install but that might be too much work for now.

the hacky way to do this would be to simply cd to the install directory for nova and apply the patch.